backend-service/src/main/java/com/smartparkinglot/backendservice/controller/DriverController.java

```diff
@@ -37,13 +39,15 @@
         return new ResponseEntity(driverService.Login(email, password),HttpStatus.ACCEPTED);
     }
 
-    @DeleteMapping()
+    @DeleteMapping("/delete")
+    @Operation(summary = "use it for driver deletion")
     public ResponseEntity deactiveDriver(@RequestBody Driver driver){
         driverService.setDeactive(driver);
         return new ResponseEntity(HttpStatus.OK);
     }
 
-    @DeleteMapping("delete")
+    @DeleteMapping("admin-delete")
+    @Operation(summary = "DONT USE it for driver deletion")
```

Comment:
[This Comment](https://github.com/bitirme-projesi-beu/backend-service/pull/14/files#r433806525) includes this one too.